image.go

```diff
@@ -449,7 +449,15 @@ func (i *image) getLayers(ctx context.Context, manifest ocispec.Manifest) ([]roo
 	if err != nil {
 		return nil, fmt.Errorf("failed to resolve rootfs: %w", err)
 	}
-	if len(diffIDs) != len(manifest.Layers) {
+
+	// parse out the image layers from oci artifact layers
```

Comment:
This is simliar to what transfer service does: https://github.com/containerd/containerd/blob/main/pkg/unpack/unpacker.go#L188